serverless.yml

```diff
@@ -91,7 +92,8 @@ provider:
 
     - Effect: Allow
       Action:
-        - redshift:ModifyCluster
```

Comment:
Hi @bcw1000. This would break AutoRemediateRS-001 ?! Is there a reason to remove this ?

Reply:
^ Planning to re-design this part to have one IAM Role per Lambda function to a) avoid such issues b) implement the Principle Of Least Privilege and comply with https://www.cloudconformity.com/conformity-rules/Lambda/sharing-an-iam-role-within-more-than-one-lambda-function.html